File: Practica2/python/circulo.py
import turtle
from punto import Punto

class Circulo:
    def __init__(self, centro: Punto, radio: float):
        self.centro = centro
        self.radio = radio

    def dibujaCirculo(self, t):
        t.penup()
        t.goto(self.centro.x, self.centro.y - self.radio)  # Ajuste de posición
        t.pendown()
        t.circle(self.radio)

    def __str__(self):
        return f"Circulo(Centro={self.centro}, Radio={self.radio})"

File: Practica2/Java/Circulo.java
public class Circulo {
    private Punto centro;
    private float radio;

    public Circulo(Punto centro, float radio) {
        this.centro = centro;
        this.radio = radio;
    }

    public Punto getCentro() { return centro; }
    public float getRadio() { return radio; }

    @Override
    public String toString() {
        return "Círculo con centro " + centro + " y radio " + radio;
    }
}